cmd/evm/internal/t8ntool/execution.go

```diff
@@ -354,6 +361,28 @@ func (pre *Prestate) Apply(vmConfig vm.Config, chainConfig *params.ChainConfig,
 		amount := new(big.Int).Mul(new(big.Int).SetUint64(w.Amount), big.NewInt(params.GWei))
 		statedb.AddBalance(w.Address, uint256.MustFromBig(amount), tracing.BalanceIncreaseWithdrawal)
 	}
+
+	// Gather the execution-layer triggered requests.
+	var requests [][]byte
+	if chainConfig.IsPrague(vmContext.BlockNumber, vmContext.Time) {
```

Comment:
Interesting that this processing is placed _after_ the block rewards. The spec doesn't explicitly say which ("at end end of") is correct, and it doesn't matter, since neither of these syscalls are influenced by the balances. 

If they were, one could use the block reward to trigger some diff between two clients that had a different order.